Sources/PostgreSQL/Database/PostgreSQLDatabaseConfig.swift

```diff
@@ -0,0 +1,23 @@
+/// Config options for a `PostgreSQLConnection`
+public struct PostgreSQLDatabaseConfig {
+    /// Creates a `PostgreSQLDatabaseConfig` with default settings.
+    public static func `default`() -> PostgreSQLDatabaseConfig {
+        return .init(hostname: "localhost", port: 5432, username: "postgres")
+    }
+
+    /// Destination hostname.
+    public let hostname: String
+
+    /// Destination port.
+    public let port: UInt16
+
+    /// Username to authenticate.
+    public let username: String
+
+    /// Creates a new `PostgreSQLDatabaseConfig`.
+    public init(hostname: String, port: UInt16, username: String) {
```

Comment:
What about db password?

Reply:
Also, the name of the database itself.

Reply:
And the schema.

Reply:
Schema is covered with migration config right?

Reply:
Not _that_ "schema". In Postgres, every database contains [schemas](https://www.postgresql.org/docs/9.1/static/ddl-schemas.html) (the default one being `public`), which contain the tables.

It was a pretty serious limitation of the legacy postgresql package (and the provider) that there was no parameter exposed for changing the schema, so it always connected to public. Just check the issues of the repos :)

Reply:
Ahhh I had no idea!